src/web/app/pages-instructor/instructor-session-individual-extension-page/instructor-session-individual-extension-page.component.html

```diff
@@ -0,0 +1,208 @@
+<tm-loading-retry [shouldShowRetry]="hasLoadingFeedbackSessionFailed"
+  [message]="'Failed to load course and feedback session'" (retryEvent)="loadFeedbackSessionAndIndividuals()">
+  <div *tmIsLoading="isLoadingFeedbackSession">
+    <div class="card bg-light top-padded text-center text-sm-left">
+      <div class="card-body">
+        <div class="row text-center">
+          <div class="col-md-2 text-md-right font-bold">Course ID</div>
+          <div id="time-zone" class="col-md-3 text-md-left">
+            {{ courseId }}
+          </div>
+        </div>
+        <div class="row text-center">
+          <div class="col-md-2 text-md-right font-bold">Time Zone</div>
+          <div id="time-zone" class="col-md-3 text-md-left">
+            {{ feedbackSessionTimeZone }}
+          </div>
+        </div>
+        <div class="row text-center">
+          <div class="col-md-2 text-md-right font-bold">Course Name</div>
+          <div id="course-name" class="col-md-10 text-md-left">
+            {{ courseName }}
+          </div>
+        </div>
+        <div class="row text-center">
+          <div class="col-md-2 text-md-right font-bold">Session Name</div>
+          <div id="course-name" class="col-md-10 text-md-left">
+            {{ feedbackSessionName }}
+          </div>
+        </div>
+        <div class="row text-center">
+          <div class="col-md-2 text-md-right font-bold">Original Deadline</div>
+          <div id="course-name" class="col-md-10 text-md-left">
+            {{ feedbackSessionEndingTime | formatDateDetail: feedbackSessionTimeZone }}
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="container ml-1">
```

Comment:
One last minor UI nit, 
<img width="1357" alt="Screen Shot 2022-03-30 at 8 05 15 PM" src="https://user-images.githubusercontent.com/60355570/160830594-610dc440-aafa-4792-a93c-3fca99c65baa.png">

Can remove the margin around the student/instructor tables and the text? Unless this is intentional, but I doubt so.

```suggestion
    <div>
```